presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveRedirectionToIceberg.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.assertj.core.api.Condition;
+import org.testng.annotations.Test;
+
+import java.sql.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tests.TestGroups.HIVE_REDIRECTION_TO_ICEBERG;
+import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class TestHiveRedirectionToIceberg
```

Comment:
❤️  for a product test!

Did you create or plan to create a QueryRunner-based one too?
That wouldn't make the PT obsolete, but could be helpful for fine-grained testing or do debug things, should anything ever malfunction.

Reply:
Didn't create a QueryRunner-based one because we'll need to install both the Iceberg and Hive plugins in one query runner.

Reply:
Can you use both from `presto-tests`?